framework/include/dgkernels/DGKernel.h

```diff
@@ -222,7 +222,7 @@ class DGKernel : public MooseObject,
   // number)
   static const BoundaryID InternalBndId;
 
-private:
+protected:
```

Comment:
Just making sure. It's not wrong to use these same statics in two different classes but you'll want to verify that we don't evaluate DGKernels and Interface kernels at the same time (pretty sure we don't) or you might have a little bit of lock contention.

Reply:
I want to make sure I'm understanding things. We have these locks to make sure that we don't have a race condition where multiple threads are trying to set the aux variable solution at the same time right? In the context of these save-in variables, you could be saving into the same variable from multiple kernel, dgkernel, bc, interface kernel objects. Wouldn't you want lock contention in that case? E.g. if a dgkernel was trying to add to the variable solution at the same time as an interface kernel was trying to add-in, wouldn't you want the lock in place?

Reply:
You've got it. If these are indeed the same vectors (as you just convinced me that they are), then we should be using the same locks. This is fine.